Cargo.toml

```diff
@@ -22,3 +22,5 @@ glob = "0.3"
 toml = "0.5"
 clap = "2.32"
 once_cell = "1.5"
+md5 = "0.7"
```

Comment:
Please consider using a contemporary hash function, e.g. the [`blake3`](https://crates.io/crates/blake3) crate provides a faster hash with better statistical properties. It even supports hex formatting out of the box...